src/datachannel.html

```diff
@@ -45,7 +44,7 @@ <h3>Usage</h3>
       sendMessageBtn.addEventListener('click', async () => {
         data = messageElement.value;
 
-        participantAddress = dropDownListElement.options[dropDownListElement.selectedIndex].value;
+        participantAddress = dropDownListElement.value;
 
         // If no particular address is selected, send to all participants.
         if (participantAddress === 'default') {
```

Comment:
In the example I just wrote, the "all" value is "" instead of "default" and therefore you would then not need this if statement. You could just use:
await room.sendMessageToParticipant(data, participantAddress);
see https://github.com/ATOS-VIRTUAL-CARE/vcs-realtime-sdk-web-demo/commit/a18e3185e6e64fe7a99f037d811fd1ae0f6fb639#diff-eda7f472e043fedce02f08e273e9b91925c3396d393d4a168395f7faf94b20f5R9